server/bleep/src/indexes/file.rs

```diff
@@ -59,7 +59,13 @@ impl<'a> Workload<'a> {
             hash.update(
                 self.file_filter
                     .is_allowed(&self.relative_path)
-                    .map(|_| &b"__filter_override"[..])
+                    .map(|include| {
+                        if include {
+                            &b"__filter_override_include"[..]
+                        } else {
+                            &b"__filter_override_exclude"[..]
+                        }
+                    })
                     .unwrap_or(&b"__no_filter_override"[..]),
```

Comment:
`hash.update(b"foobar")` should work too, without the conversion of the array to `&[u8]`.

Reply:
ah i suppose the `unwrap_or` will not typecheck without slices.